etc/caddy/Caddyfile

```diff
@@ -13,6 +13,11 @@ http://:80 {
     root * /var/www/datasets
   }
 
+  # traffic data
+  handle_path /traffic/* {
```

Comment:
More traffic related publications like e.g. constructions and routing strategies may follow. So could we use a more specific path, e.g. `/traffic/incidents-bw/`.

And I suggest to publish static / periodically updated feeds under `/datasets/traffic/incidents-bw/`